.github/workflows/release.yml

```diff
@@ -0,0 +1,50 @@
+on:
+  push:
+    tags:
+      - '*'
+
+name: Upload Release Asset and Deploy to Nexus
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 17
```

Comment:
Is 17 intended? We're using 8. I know, it sucks.

Reply:
Yes intended. Don't like using an old jdk, when the maven compile target level is defined anyway.